haskell/src/Stablecoin/Util.hs

```diff
@@ -0,0 +1,12 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: MIT
+
+module Stablecoin.Util
+  ( aesonOptions
+  ) where
+
+import Data.Aeson
```

Comment:
Shoulnd't we use qualified imports here?

Reply:
Done